conda_build/build.py

```diff
@@ -500,6 +501,8 @@ def test(m, verbose=True, channel_urls=(), override_channels=False):
 
     tmp_dir = join(config.croot, 'test-tmp_dir')
     rm_rf(tmp_dir)
+    time.sleep(2)  # give the OS some time to complete the rm_rf() command
```

Comment:
Here is the error message:
```
BUILD END: use_foo-1.0-0
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda-build/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "C:\conda\Scripts\conda-build-script.py", line 4, in <module>
    sys.exit(main())
  File "C:\conda\lib\site-packages\conda_build\main_build.py", line 190, in main
    args_func(args, p)
  File "C:\conda\lib\site-packages\conda_build\main_build.py", line 432, in args_func
    args.func(args, p)
  File "C:\conda\lib\site-packages\conda_build\main_build.py", line 419, in execute
    channel_urls=channel_urls, override_channels=args.override_channels)
  File "C:\conda\lib\site-packages\conda_build\build.py", line 506, in test
    os.makedirs(tmp_dir)
  File "C:\conda\lib\os.py", line 157, in makedirs
    mkdir(name, mode)
WindowsError: [Error 5] Access denied: 'C:\\conda\\conda-bld\\test-tmp_dir'

> conda info
Current conda install:

             platform : win-64
        conda version : 3.16.0-36-gd28d3f3-dirty
  conda-build version : 1.17.0+49.g0495f64
       python version : 2.7.9.final.0
     requests version : 2.7.0
     root environment : C:\conda  (writable)
  default environment : C:\conda\envs\foo
     envs directories : C:\conda\envs
        package cache : C:\conda\pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
    is foreign system : False
```
The error only occurs with `test-tmp_dir`. Moreover, it seems that it only happens when the package doesn't define tests, but I'm not sure.